docs/guides/high-availability/concepts-threads.adoc

```diff
@@ -20,18 +20,19 @@ For a configuration where this is applied, visit <@links.ha id="deploy-keycloak-
 
 The Quarkus executor thread pool is configured in https://quarkus.io/guides/all-config#quarkus-core_quarkus.thread-pool.max-threads[`quarkus.thread-pool.max-threads`] and has a maximum size of at least 200 threads.
 Depending on the available CPU cores, it can grow even larger.
-Threads are created as needed, and will end when no longer needed, so the system will scale up and down as needed.
+Threads are created as needed, and will end when no longer needed, so the system will scale up and down automatically.
 
-When running on Kubernetes, adjust the number of worker threads to avoid creating more load than what the CPU limit for the Pod to avoid throttling, which would lead to congestion.
+When running on Kubernetes, adjust the number of worker threads to avoid creating more load than what the CPU limit allows for the Pod to avoid throttling, which would lead to congestion.
 When running on physical machines, adjust the number of worker threads to avoid creating more load than the node can handle to avoid congestion.
 Congestion would result in longer response times and an increased memory usage, and eventually an unstable system.
 
 Ideally, you should start with a low limit of threads and adjust it accordingly to the target throughput and response time.
-When the load and the number of threads increases, the bottleneck can become be the database connections.
+When the load and the number of threads increases, the database connections can also become a bottleneck.
 Once a request cannot acquire a database connection within 5 seconds, it will fail with a message in the log like `Unable to acquire JDBC Connection`.
 The caller will receive a response with a 5xx HTTP status code indicating a server side error.
 
-If you increase the number of database connections and the number of threads too much, the system will be congested under a high load with requests queueing up and all callers getting a bad performance.
+If you increase the number of database connections and the number of threads too much, the system will be congested under a high load with requests queueing up and all callers, which leads to bad performance.
+The number of database connections is configured via the https://quarkus.io/guides/all-config#quarkus-agroal_quarkus-agroal-agroal-database-connection-pool[`Agroal - Database connection pool`] settings `quarkus.datasource.jdbc.initial-size`, `quarkus.datasource.jdbc.min-size` and `quarkus.datasource.jdbc.max-size` respectively.
```

Comment:
Th guide should refer to Keycloak's configuration options, not the Quarkus options. See https://www.keycloak.org/server/all-config#category-database for a list of Keycloak's database options.

The rationale around this: For all supported way of configuring Keycloak, there should be Keycloak configuration options. Those usually work as a facade to Quarkus and other underlying technologies. This allows us to have Keycloak specific defaults, and sometimes also hide the complexity of a underlying technolgoies.

Reply:
Thanks for the review, I'll link to the keycloak database settings instead.